Changelogs: Added Summit cycles (#610)
seqeralabs/docs: changelog/seqera-cloud/v25.1.0_cycle11.md

---
title: Seqera Cloud v25.1.0_cycle10 - cycle12
date: 2025-05-06
tags: [seqera cloud]
---

> This changelog includes releases

Reviewer: I think we can move this list to the end of the release notes as it's a bit long

-  v25.1.0-cycle10_37f83e6
-  v25.1.0-cycle11_cde3ebf
-  v25.1.0-cycle11_0ff195a
-  v25.1.0-cycle12_3d9ec0c
-  v25.1.0-cycle12_8bf53cd
-  v25.1.0-cycle12_c98e9a5
-  v25.1.0-cycle12_1d46e0f
-  v25.1.0-cycle12_61865f4
-  v25.1.0-cycle12_fbec4ef
-  v25.1.0-cycle12_20df528

### New features and improvements

- **Fusion**
  - Add Fusion snapshots to AWS Batch.
  - Enable `DRAINING` status for Spot instance reclamation. When an Instance status is DRAINING, no new task can be scheduled on it. More info on this behaviour can be found at [https://docs.aws.amazon.com/AmazonECS/latest/developerguide/spot-instance-draining-linux-container.html](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/spot-instance-draining-linux-container.html).

>   - AWS Cloud compute environment defaults to the public Seqera AMI if no AMI ID is specified during compute environment creation.

Reviewer: Need to describe what AWS Cloud compute environments are a bit more

- **Studios**
  - Add Spot option for AWS single VM compute environment.
- Support markdown rendering for Nextflow object field `help_text` and `description` properties.

> - New Seqera Platform sidebar which replaces the top navigation.

Reviewer 2: ```suggestion
- New sidebar, replacing top navigation.
```

> - Run details page updates and improvements.

Reviewer 2: ```suggestion
- New [Run details](https://docs.seqera.io/platform-cloud/monitoring/run-details) page design. 
```

  - Added **Containers** tab to task details page.

> - Single VM AWS compute environment.

Reviewer: This is the AWS Cloud CE – We should combine

date: 2025-05-06
tags: [seqera cloud]
---

### New features and improvements

## Seqera Compute

[Seqera Compute](https://docs.seqera.io/platform-cloud/compute-envs/seqera-compute) is now in public preview. Seqera Compute is a fully managed, optimized compute environment which eliminates infrastructure complexity by automating environment setup, resource provisioning, scaling, and optimization. [Manage your resources and costs effectively with Seqera Compute credits](https://docs.seqera.io/platform-cloud/compute-envs/seqera-compute#manage-seqera-compute-credits), launch .

## Platform interface updates

### Pipeline run details

The [pipelines run details](https://docs.seqera.io/platform-cloud/monitoring/run-details#tasks) view has been refreshed and includes improvements such as:

- Clear, dynamic view of pipeline execution in real time via a progress bar.
- A dedicated **Run info** tab with a centralized view of run metadata and status details.
- Improved navigation tabs for **Tasks**, **Logs**, **Metrics**, and **Containers**.

### New Seqera Platform sidebar

The Platform top navigation has been replaced with a sidebar for improved navigation. 

> ## Single VM AWS compute environment

Author: ```suggestion
## AWS Cloud compute environment
```

> The [new single VM AWS compute environment](https://docs.seqera.io/platform-cloud/compute-envs/aws-cloud) supports faster startup time, simplified configuration, and fewer AWS dependencies, as well as allowing Studios to launch on Spot instances. This type of compute environment is best suited to run Studios and small to medium-sized pipelines. It offers more predictable compute pricing, given the fixed instance type. It spins up a standalone EC2 instance and executes a Nextflow pipeline or Studio session with a local executor on the EC2 machine. At the end of the execution, the instance is terminated.

Author: ```suggestion
The [new AWS Cloud compute environment](https://docs.seqera.io/platform-cloud/compute-envs/aws-cloud) supports faster startup time, simplified configuration, and fewer AWS dependencies, as well as allowing Studios to launch on Spot instances. This type of compute environment is best suited to run Studios and small to medium-sized pipelines. It offers more predictable compute pricing, given the fixed instance type. It spins up a standalone EC2 instance and executes a Nextflow pipeline or Studio session with a local executor on the EC2 machine. At the end of the execution, the instance is terminated.
```

---
title: Seqera Cloud v25.1.0_cycle10 - cycle12
date: 2025-05-06
tags: [seqera cloud]
---

## Seqera Compute

> [Seqera Compute](https://docs.seqera.io/platform-cloud/compute-envs/seqera-compute) is now in public preview. Seqera Compute is a fully managed, optimized compute environment which eliminates infrastructure complexity by automating environment setup, resource provisioning, scaling, and optimization. [Manage your resources and costs effectively with Seqera Compute credits](https://docs.seqera.io/platform-cloud/compute-envs/seqera-compute#manage-seqera-compute-credits), launch .

Author: ```suggestion
[Seqera Compute](https://docs.seqera.io/platform-cloud/compute-envs/seqera-compute) is now in public preview. Seqera Compute is a fully managed, optimized compute environment which eliminates infrastructure complexity by automating environment setup, resource provisioning, scaling, and optimization.

## Platform interface updates

### Pipeline run details

> The [pipelines run details](https://docs.seqera.io/platform-cloud/monitoring/run-details#tasks) view has been refreshed and includes improvements such as:

Reviewer 2: ```suggestion
The [pipelines run details](https://docs.seqera.io/platform-cloud/monitoring/run-details) view has been refreshed and includes improvements such as:
```

- Clear, dynamic view of pipeline execution in real time via a progress bar.
- A dedicated **Run info** tab with a centralized view of run metadata and status details.
- Improved navigation tabs for **Tasks**, **Logs**, **Metrics**, and **Containers**.

### New Seqera Platform sidebar

The Platform top navigation has been replaced with a sidebar for improved navigation. 

## AWS Cloud compute environment

The [new AWS Cloud compute environment](https://docs.seqera.io/platform-cloud/compute-envs/aws-cloud) supports faster startup time, simplified configuration, and fewer AWS dependencies, as well as allowing Studios to launch on Spot instances. This type of compute environment is best suited to run Studios and small to medium-sized pipelines. It offers more predictable compute pricing, given the fixed instance type. It spins up a standalone EC2 instance and executes a Nextflow pipeline or Studio session with a local executor on the EC2 machine. At the end of the execution, the instance is terminated.

## API documentation improvements

> The [Seqera Platform API documentation](https://docs.seqera.io/platform-api/seqera-api) has been updated is now broken down by feature for easier reference, with detailed and interactive parameter examples.  

Reviewer 2: ```suggestion
The [Seqera Platform API documentation](https://docs.seqera.io/platform-api/seqera-api) has been updated. Featuring overview pages with parameter and request body tables, categorized by Platform entity (compute environments, pipelines, etc.), the documentation now provides easier reference and detailed request examples.
```

## Other new features and improvements

- **Fusion**
  - Add Fusion snapshots to AWS Batch.

>   - Enable `DRAINING` status for Spot instance reclamation. When an Instance status is DRAINING, no new task can be scheduled on it. See [Configuring Amazon ECS Linux container instances to receive Spot Instance notices](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/spot-instance-draining-linux-container.html) for more information.

Reviewer 2: ```suggestion
  - Enable `DRAINING` status for Spot instance reclamation. When an instance is in DRAINING status, no new task can be scheduled on it. See [Configuring Amazon ECS Linux container instances to receive Spot Instance notices](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/spot-instance-draining-linux-container.html) for more information.
```

  - AWS Cloud compute environment defaults to the public Seqera AMI if no AMI ID is specified during compute environment creation.
- **Studios**

>   - Add Spot option for AWS single VM compute environment.

Reviewer 2: ```suggestion
  - Add Spot option for AWS Cloud compute environment.
```

>   - Enable `DRAINING` status for Spot instance reclamation. When an Instance status is DRAINING, no new task can be scheduled on it. More info on this behaviour can be found at [https://docs.aws.amazon.com/AmazonECS/latest/developerguide/spot-instance-draining-linux-container.html](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/spot-instance-draining-linux-container.html).

Author: ```suggestion
  - Enable `DRAINING` status for Spot instance reclamation. When an Instance status is DRAINING, no new task can be scheduled on it. See [Configuring Amazon ECS Linux container instances to receive Spot Instance notices](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/spot-instance-draining-linux-container.html) for more information.
```
